packages/now-next/test/fixtures/00-optional-fallback-revalidate/now.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Let's start using `vercel.json` for new tests (the test harness should already support this I believe)